lib/api/controllers/securityController.js

```diff
@@ -105,7 +125,7 @@ class SecurityController {
     return this.kuzzle.repositories.role.loadRole(request.input.resource._id)
       .then(role => {
         if (!role) {
-          return Promise.reject(new NotFoundError(`Role with id ${request.input.resource._id} not found`));
+          return Bluebird.reject(new NotFoundError(`Role with id ${request.input.resource._id} not found`));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 108 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)